packages/instant/src/components/erc20_token_selector.tsx

```diff
@@ -123,10 +122,20 @@ interface TokenSelectorRowIconProps {
     token: ERC20Asset;
 }
 
+const getTokenIcon = (symbol: string): React.StatelessComponent | undefined => {
+    try {
+        return require(`../assets/icons/${symbol}.svg`) as React.StatelessComponent;
```

Comment:
hmm dunno if we have to go as far as casting this as a stateless component

Reply:
i think it's unlikely that a required svg would have state, and I like the typesafety of the null checking, but I can change back to `any` if you feel strongly

Reply:
I've always thought there should be a separate file where we iterate over all the token symbols in our data and require the icon. This also ensures that we (the developers) can't include a new token without an svg.

Reply:
I think this would be a good CI task